deploy-edgecluster/verify.sh

```diff
@@ -35,6 +35,15 @@ function check_aci() {
         exit 1
     else
         echo "AgentClusterInstall for ${cluster} verified"
+        oc --kubeconfig=${KUBECONFIG_HUB} get -n ${cluster} secret/${cluster}-keypair >/dev/null 2>&1
+        if [ $? -eq 0 ]; then
+            echo "The RSA key for ${cluster} is already stored on the hub"
+        else
```

Comment:
fwiw, I did think of this in the previous PR but I was assuming that it'd have been created anyways. Or rather, I figured that the `deploy.sh` script in the `finish-deployment` would take care of everything.

I like what you did here better as we upload the keys the moment they are created. We should do the same with the edge cluster, fwiw.

Reply:
Ack,
IMHO, the edge cluster portion can be done as part of refactoring, i.e. in a separate MR. I didn't see (yet) a demand to have the key stored on edge for the pipeline to finish.